thinc/util.py

```diff
@@ -351,12 +392,25 @@ def xp2torch(
     return torch_tensor
 
 
-def torch2xp(torch_tensor: "torch.Tensor") -> ArrayXd:  # pragma: no cover
-    """Convert a torch tensor to a numpy or cupy tensor."""
-    if torch_tensor.is_cuda:
-        return cupy.fromDlpack(torch.utils.dlpack.to_dlpack(torch_tensor))
+def torch2xp(
+    torch_tensor: "torch.Tensor", *, ops: Any = None
```

Comment:
```suggestion
    torch_tensor: "torch.Tensor", *, ops: Ops = None
```

or if that leads to a recursive import (probably does), then:

```suggestion
    torch_tensor: "torch.Tensor", *, ops: "Ops" = None
```

(I think)